Core/Utilities/TimeLeft/MJFTimeLeft.py

```diff
@@ -73,16 +73,14 @@ def getResourceUsage( self ):
       cpuLimit = int( urllib.urlopen(jobFeaturesPath + '/cpu_limit_secs').read() )
     except:
       self.log.warn( 'Could not determine cpu limit from $JOBFEATURES/cpu_limit_secs' )
-      cpuLimit = wallClockLimit
 
     wallClock = int(time.time()) - jobStartSecs
-    # We cannot get CPU usage from MJF, so for now use wallClock figure
-    cpu = wallClock
+    # We cannot get CPU usage from MJF
       
     consumed = {'CPU':cpu, 'CPULimit':cpuLimit, 'WallClock':wallClock, 'WallClockLimit':wallClockLimit}
     self.log.debug( consumed )
```

Comment:
This one is not on my fault! You should blame 254cf51a instead ;)

More seriously, I fix in 7ccdefa